src/components/molecules/product-card/components/UnavailabilityMessage.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+export const UnavailabilityMessage = () => {
+  const Message = styled.div`
+    background-image: linear-gradient(to right, #aa4931, #d1583a);
+    box-shadow: 0px 0px 5px #2d2d2d;
+    color: #ededed;
+    display: flex;
+    font: 400 1em system-ui;
+    font-style: italic;
+    height: 10%;
+    justify-content: center;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 60%;
+    padding-top: 0.2em;
+    text-align: center;
+    text-shadow: 1px 0px 0px #2d2d2d;
+    transition: 0.3s all;
+    width: 50%;
+
+    @media screen and (min-width: 1024px) {
+      opacity: 0;
+    }
+  `;
+
+  return (
+    <Message className="c-product-card__unavailability-message">
```

Comment:
why do we use emotion, only to give it classes anyway 🤔 (i dont know emotion that well 🤷‍♂)

Reply:
This seemed like the best way to create the hover effects controlled by the container element, open to other suggestions tho